pkg/sql/drop_test.go

```diff
@@ -1380,7 +1380,7 @@ CREATE TABLE t.child(k INT PRIMARY KEY REFERENCES t.parent);
 	// for updating the referenced table has an empty description.
 	testutils.SucceedsSoon(t, func() error {
 		var count int
-		sqlRun.QueryRow(t, `SELECT count(*) FROM [SHOW JOBS] WHERE description = 'GC for ' AND status = 'succeeded'`).Scan(&count)
+		sqlRun.QueryRow(t, `SELECT count(*) FROM [SHOW JOBS] WHERE description LIKE 'removing outbound FK %' AND status = 'succeeded'`).Scan(&count)
```

Comment:
I didn't notice this the first time around but I think you need to add a `job_type = 'SCHEMA CHANGE GC'` to your `WHERE` clause, and update the comment three lines above which is now out of date.